.github/workflows/maven-verify.yml

```diff
@@ -28,7 +28,9 @@ jobs:
     with:
       # maven-args: "-Dinvoker.parallelThreads=2" cannot do this as this generate some concurrent download issues
       jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "adopt-openj9" ]'
-      ff-jdk: "20"
-      jdk-matrix: '[ "8", "11", "17", "20" ]'
-      matrix-exclude: '[{ "jdk": "8", "distribution": "microsoft"},{ "jdk": "20", "distribution": "microsoft"},{ "jdk": "20", "distribution": "adopt-openj9"},]'
-      maven-matrix: '[ "3.6.3", "3.8.8", "3.9.5" ]'  # Maven versions matrix for verify builds
+      matrix-exclude: '[
+          { "jdk": "8", "distribution": "microsoft" },
```

Comment:
Why is this one excluded? It's not a Mac

Reply:
It has been excluded early ... probably microsoft distribution does not support jdk 8 at all